### ISSUE TITLE
npm build seems broken

### ISSUE THREAD

**Issue by vhpoet** [typicode/lowdb#116]
I can't use options.format with the current npm build. 

```
ReferenceError: options is not defined
```

Here's the compiled `_index.js`

```
if (format) {
      var _options = options;
      var _format = _options.format;

      db.serialize = _format.serialize;
      db.deserialize = _format.deserialize;
    }
```

**Comment by typicode** [comment]
Thank you for the report! Will check that today.
I'll add also a failing test to avoid it happening again.

**Comment by vhpoet** [comment]
@typicode don't have time to wait, so just made a PR haha. If you could merge it and release a new version today, you would be my hero!

**Comment by typicode** [comment]
thanks :)

**Comment by typicode** [comment]
Published `v0.13.1`
